doc/modules/multiclass.rst

```diff
@@ -64,20 +64,69 @@ Below is a summary of the classifiers supported by scikit-learn
 grouped by strategy; you don't need the meta-estimators in this class
 if you're using one of these, unless you want custom multiclass behavior:
 
-  - Inherently multiclass: :ref:`Naive Bayes <naive_bayes>`,
-    :ref:`LDA and QDA <lda_qda>`,
-    :ref:`Decision Trees <tree>`, :ref:`Random Forests <forest>`,
-    :ref:`Nearest Neighbors <neighbors>`,
-    setting ``multi_class='multinomial'`` in
-    :class:`sklearn.linear_model.LogisticRegression`.
-  - Support multilabel: :ref:`Decision Trees <tree>`,
-    :ref:`Random Forests <forest>`, :ref:`Nearest Neighbors <neighbors>`.
-  - One-Vs-One: :class:`sklearn.svm.SVC`.
-  - One-Vs-All: all linear models except :class:`sklearn.svm.SVC`.
-
-Some estimators also support multioutput-multiclass classification
-tasks :ref:`Decision Trees <tree>`, :ref:`Random Forests <forest>`,
-:ref:`Nearest Neighbors <neighbors>`.
+  - Inherently multiclass:
+
+    - :class:`sklearn.naive_bayes.BernoulliNB`
+    - :class:`sklearn.tree.DecisionTreeClassifier`
+    - :class:`sklearn.tree.ExtraTreeClassifier`
+    - :class:`sklearn.ensemble.ExtraTreesClassifier`
+    - :class:`sklearn.naive_bayes.GaussianNB`
+    - :class:`sklearn.neighbors.KNeighborsClassifier`
+    - :class:`sklearn.semi_supervised.LabelPropagation`
+    - :class:`sklearn.semi_supervised.LabelSpreading`
+    - :class:`sklearn.discriminant_analysis.LinearDiscriminantAnalysis`
+    - :class:`sklearn.svm.LinearSVC` (setting multi_class="crammer_singer")
+    - :class:`sklearn.linear_model.LogisticRegression` (setting multi_class="multinomial")
+    - :class:`sklearn.linear_model.LogisticRegressionCV` (setting multi_class="multinomial")
+    - :class:`sklearn.neural_network.MLPClassifier`
+    - :class:`sklearn.neighbors.NearestCentroid`
+    - :class:`sklearn.linear_model.PassiveAggressiveClassifier`
+    - :class:`sklearn.linear_model.Perceptron`
+    - :class:`sklearn.discriminant_analysis.QuadraticDiscriminantAnalysis`
+    - :class:`sklearn.neighbors.RadiusNeighborsClassifier`
+    - :class:`sklearn.ensemble.RandomForestClassifier`
+    - :class:`sklearn.linear_model.RidgeClassifier`
```

Comment:
I'm ambivalent what to say about RidgeClassifier. It doesn't really have a classification loss at all. I guess you could call it inherently multi-class and it's not really wrong.